vnext/ReactUWP/Views/ReactControl.cpp

```diff
@@ -128,6 +134,8 @@ void ReactControl::AttachRoot() noexcept
 #endif
 
   m_touchEventHandler->AddTouchHandlers(m_xamlRootView);
+  m_previewKeyboardEventHandlerOnRoot->hook(m_xamlRootView);
```

Comment:
> m_previewKeyboardEventHandlerOnRoot->hook(m_xamlRootView); [](start = 1, length = 59)

For pointer events we have to AddTouchHandlers in the FlyoutViewManager and the PopupViewManager -- do we need to do the same with the keyboard event handler?

Reply:
Good call. I did testing and it doesn't work for flyout and popup.
Now I add them back just as touchEventHandler.